infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/types/SerializableObjectTypeDefinition.java

```diff
@@ -25,7 +25,7 @@
 class SerializableObjectTypeDefinition<TObject> implements SerializableTypeDefinition<TObject> {
 
   private final Optional<String> name;
-  private final Map<String, FieldDefinition<TObject>> fields;
+  final Map<String, FieldDefinition<TObject>> fields;
```

Comment:
nit: Probably protected if we're aiming for this to be available to subclasses or possibly not needed in this PR.